dashboard/src/t5gweb/static/js/table.js

```diff
@@ -120,6 +120,7 @@ $(document).ready(function () {
       [2, "desc"],
       [3, "desc"],
       [4, "desc"],
+      [5, "desc"],
```

Comment:
This is linked to the Summary Column, don't think we need to restore this. It sorts the summary column in alphabetical order
![image](https://github.com/user-attachments/assets/307e8360-7b82-4522-b5cc-9091c1ffa2b8)